### PR TITLE
ImEditor

### PR DESCRIPTION
**Pull request by Huluti**
I'd like to submit ImEditor to flathub.

https://github.com/ImEditor/ImEditor
https://imeditor.github.io/

### CODE REVIEW

File: io.github.ImEditor.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options": {
```
Comment (bilelmoussaoui) [inline io.github.ImEditor.json]: You can remove those build-options. They are the default now.

File: io.github.ImEditor.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/51/fe/18125dc680720e4c3086dd3f5f95d80057c41ab98326877fc7d3ff6d0ee5/Pillow-6.1.0.tar.gz",
+                    "sha256": "0804f77cb1e9b6dbd37601cee11283bba39a8d44b9ddb053400c58e0c0d7d9de"
+                }
+            ]
+        },
+        {
+            "name": "imeditor",
+            "builddir": true,
```
Comment (bilelmoussaoui) [inline io.github.ImEditor.json]:
```suggestion

```

**Comment by bilelmoussaoui** [comment]
bot, build io.github.ImEditor

**Comment by nedrichards** [comment]
bot, build io.github.ImEditor

**Comment by LeandroStanger** [comment]
bot, build io.github.ImEditor

**Comment by nedrichards** [comment]
@Huluti fwiw you can run these commands too when you make updates

**Comment by Huluti** [comment]
@ nedrichards @LeandroStanger can I base this MR on the latest commit?
https://github.com/ImEditor/ImEditor/commit/bc2a0f0a7b926fa2abbd5080278d4899d4032e47

**Comment by Huluti** [comment]
@LeandroStanger wihout re-do a release?

**Comment by Huluti** [comment]
bot, build io.github.ImEditor

**Comment by Huluti** [comment]
bot, build io.github.ImEditor

**Comment by nedrichards** [comment]
@Huluti you should use the release and extract and apply that commit as a patch - just like this recent app. https://github.com/flathub/im.kaidan.kaidan/blob/master/im.kaidan.kaidan.json#L59

then on the next release you can drop the patches as they'll be included.

**Comment by nedrichards** [comment]
In order to safely escape the sandbox to open/save new files you need to make sure you're using https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html That automatically takes care of using the flatpak portal for you.

**Comment by Huluti** [comment]
bot, build io.github.ImEditor

**Comment by Huluti** [comment]
bot, build io.github.ImEditor

**Comment by Huluti** [comment]
> In order to safely escape the sandbox to open/save new files you need to make sure you're using https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html That automatically takes care of using the flatpak portal for you.

@nedrichards Should be ok now!

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.github.ImEditor

**Pull request closed without merging**